services/monolith/workspace/lib/sms/mock_adapter.rb

```
@@ -0,0 +1,51 @@
+  # Logs messages to stdout instead of sending real SMS.
+  #
+  # Usage:
+  #   adapter = SMS::MockAdapter.new
+  #   adapter.send_verification(phone_number: "+81901234567", code: "1234")
+  #
+  class MockAdapter < Service
+    attr_reader :sent_messages
+
+    def initialize
+      @sent_messages = []
+    end
```

RuboCop (1.84.0)

[warning] Call `super` to initialize state of the parent class.

(Lint/MissingSuper)